app/models/assessment.rb

```diff
@@ -1,16 +1,22 @@
 require 'time'
 
 class Assessment < FormResponses
-  def initialize(constructor = nil)
-    super(constructor)
-    self['date_created'] = Time.now.utc.iso8601
+  ATTRS = %i[id date_created confidence_level_required
+             evidence_id_list
+             identity_exists_over_time identity_with_organisation_proved org_check_method org_activity_found activity_time_period
+             identity_been_stolen_or_used_fraudulently check_identity_not_stolen_or_used_fraudulently counter_fraud_data_sources idependent_counter_fraud_sources].freeze
```

Comment:
~idependent~  => `independent` ?

Reply:
Yeah, the typo was already there so I was thinking about fixing that in a separate PR :-)